openapi/openapi.yaml

```
@@ -390,7 +390,8 @@ tags:
       An order generates an invoice.
       A subscription order generates an invoice for each service period.
       For more information, see [Orders](https://www.rebilly.com/docs/dev-docs/concepts/#orders).
-
+  - name: Orders experimental
+    description: Use these operations to manage orders with mixed recurring plan subscriptions.
```

suggestion: If possible, it would be good to add more detail to the description so that the reader is clear on what these operations are for

I'm not sure if I have changed the meaning here. Please feel free to edit, or to tag me for input

```suggestion
    description: Use these operations to manage orders that include items with mixed recurring subscription plans. For example, an order that includes 3 subscriptions plans that all have different service periods.
```

---

Let's proceed without example for the time being like:

```suggestion
    description: Use these operations to manage orders that include items with mixed recurring subscription plans.
```

What would you say @noon-dawg?

---

Looks good @nanobash!